src/main/kotlin/net/casual/arcade/border/ArcadeBorder.kt

```diff
@@ -36,23 +43,37 @@ abstract class ArcadeBorder: WorldBorder() {
     }
 
     override fun getCenterX(): Double {
-        return this.centerX
+        return if (this.centerState != null) this.centerState.getCenter().x else 0.0
```

Comment:
`centerState` is never null
In kotlin a variable/field can never be null unless marked nullable with `?`